src/components/Calculator.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Button from './Button';
+import Result from './Result';
+import '../App.css';
+
+function Calculator() {
+  const regularButton = 'box-border border-2 border-neutral-400 col-span-1 bg-indigo-500 flex justify-center items-center';
+  const wideButton = 'box-border border-2 border-neutral-400 col-span-2 bg-indigo-500 flex justify-center items-center';
+  const display = 'box-border border-2 border-neutral-400 col-span-4 bg-indigo-500 flex justify-center items-center text-white';
+  const actionColor = 'bg-orange-500';
+  const numberColor = 'bg-stone-200';
+  const displayColor = 'bg-neutral-800';
+  const calculatorStyle = 'grid grid-cols-4 w-80 h-80 box-border border-4 border-neutral-400';
```

Comment:
- [ ] [Optional] It would be better professionally if you style your components using `css module` method instead of what you used here.